src/isl.rs

```diff
@@ -232,61 +235,61 @@ impl IslTypeRef {
 #[cfg(test)]
 mod isl_tests {
     use super::*;
-    use ion_rs::value::owned::{text_token, OwnedElement, OwnedStruct};
-    use ion_rs::value::Builder;
+    use ion_rs::value::owned::OwnedElement;
+    use ion_rs::value::reader::element_reader;
+    use ion_rs::value::reader::ElementReader;
     use rstest::*;
-    use std::iter::FromIterator;
+
+    // helper function to be used by isl tests
+    fn load(text: &str) -> OwnedElement {
+        element_reader()
+            .read_one(text.as_bytes())
+            .expect("parsing failed unexpectedly")
+    }
 
     #[rstest(
     isl_type1,isl_type2,
     case::type_constraint_with_anonymous_type(
-        /* For a schema with single anonymous type as below: 
-            { type: int }
-         */
-        (&OwnedStruct::from_iter(vec![("type", OwnedElement::from(text_token("int")))].into_iter())).try_into().unwrap(),
+        (&load(r#" // For a schema with single anonymous type
+            {type: int}
+        "#)).try_into().unwrap(),
```

Comment:
You can make another helper method that also takes care of the `try_into().unwrap()` on each of these.